testdata/serviceconfig.yml

```diff
@@ -32,7 +32,7 @@ rules:
     params:
       logName: "access_log"
       log:
-        logFormat: 1 # COMMON
+        descriptorName: "common"
```

Comment:
looking at the keys again, and how it plays out with UX, `descriptor_name` is really `logType`